browser/BUILD.gn

```diff
@@ -44,6 +45,8 @@ source_set("browser_process") {
     "brave_content_browser_client.h",
     "brave_local_state_prefs.cc",
     "brave_local_state_prefs.h",
+    "brave_performance_observers.cc",
```

Comment:
why is this in brave/browser vs components? If there is a good reason to keep it here it should be namespaced to match the component namespace